source/core.py

```diff
@@ -146,7 +147,6 @@ def resetConfiguration(factoryDefaults=False):
 	import brailleInput
 	import speech
 	import vision
-	import languageHandler

```

Comment:
It would be worth checking carefully if the order of importing languageHandler can change like this.

Reply:
I thought it was something worth trying to move out but perhaps something for a later day with the capacity to test it better.

Reply:
It would certainly be good to do that if we can. I'm just not really sure of the implications.